### PR TITLE
REP-4256 Adding our versioned docs to the landing page

### PR DESCRIPTION
**Pull request by adrianjgeorge**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,6 +2,23 @@
...
+
+    <section class="top-bar-section">
+        <ul class="right">
+            <li class="has-dropdown">
+                <a href="#">Documentation</a>
+                <ul class="dropdown">
+                    {% include versions.html %}
+                </ul>
+            </li>
+        </ul>
```
Comment (dmnjohns) [inline index.html]: How this becomes that top bar, I'll never know. Fancy formatting magic.

File: index.html [hunk 2]
```diff
@@ -2,6 +2,23 @@
 layout: default
 title: Repose
 ---
+<nav class="top-bar" data-topbar role="navigation">
```
Comment (adrianjgeorge) [inline index.html]: It's not. I tried doing that, but it broke the menu functionality, and it wasn't important enough to try to fix

Comment (dmnjohns) [inline index.html]: Fair enough.

**Comment by dmnjohns** [comment]
I think I have to chalk all of this up to magic.

I super do not know enough about the Foundation framework or other front-end things. If this is what was previewed, though, then that's good enough for me! 👍

**Comment by adrianjgeorge** [comment]
I can walk you through it on monday if you'd like, or we could add it to the next dev meeting.

**Pull request merged**